packages/indexer-agent/src/commands/start.ts

```diff
@@ -238,6 +245,16 @@ export const start = {
         type: 'string',
         required: false,
       })
+      .option('tap-address-book', {
+        description: 'TAP contracts address book file path',
+        type: 'string',
+        required: false,
+      })
+      .option('chain-finalize-time', {
+        description: 'The time in seconds that the chain finalizes blocks',
```

Comment:
Is time the right thing to use here? In other places (graph node for example) we have been using the number of blocks to be mined after a block before that block is considered final. This is because blockchains don't generate new blocks with high precision.

Reply:
We based this off info Tomas gave us, basically 

- the sequencer takes less than a second to order and post the tx in arbitrum
- 6-7 mins is the time the sequencer takes to post the tx in ethereum
- ~13 mins , its the finality time in ethereum post merge
So tit should be around 20 mins the finality time
We are defining it as 60 mins just to be sure
Either way ravs will be saved in the db in case we encounter any issue

Reply:
hey @Jannis is this clearing your concerns? can we solve this issue?

Reply:
@fordN @Jannis no feedback, so resolving